lib/big_brother/cluster.rb

```diff
@@ -11,7 +11,7 @@ module Type
     def initialize(name, attributes = {})
       @name = name
       @fwmark = attributes[:fwmark]
-      @cluster_mode = attributes.fetch(:backend_mode, Type::Default)
+      @cluster_mode = @backend_mode = attributes.fetch(:backend_mode, Type::Default)
```

Comment:
Not based off of what I could tell in the code path, but thought that might be worth discussion. `backend_mode` is the config option, but `cluster_mode` seems clearer.